packages/salesforcedx-vscode-core/src/commands/forceConfigSet.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+
+import {
+  EmptyParametersGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+export class ForceConfigSetExecutor extends SfdxCommandletExecutor<{}> {
+  private arg: string;
+
+  public constructor(arg: string) {
```

Comment:
Can we change `arg` to a more meaningful name e.g. username, defaultUsername, etc ?

Reply:
Yup!